src/blkdev.c

```diff
@@ -265,7 +270,8 @@ value uk_block_info(value v_block)
   const struct uk_blkdev_cap* cap = uk_blkdev_capabilities(block->dev);
 
   v_result = caml_alloc(3, 0);
-  Store_field(v_result, 0, Val_true);                      // ready_write
+  // ready_write
```

Comment:
```suggestion
  // read_write
```

Reply:
Note: `read_write` is currently not used in `block.ml` (or in this file) except for `get_info`. I suggest you add a check to `write` so if it's called on a read-only block, the call fails early with a clear error rather than deep in the unikraft call stack

Reply:
the built-in `Is_read_only` variant from `Mirage_block.write_error` can be used for that

Reply:
Good point, and thanks for the pointers! Implemented with [be5513e](https://github.com/fabbing/mirage-block-unikraft/pull/2/commits/be5513ef09f0417a56b72e69ceef771cbdade533).